migrations/lib/converters/discourse/steps/topics.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Migrations::Converters::Discourse
+  class Topics < ::Migrations::Converters::Base::ProgressStep
+    attr_accessor :source_db
+
+    def max_progress
+      @source_db.count <<~SQL
+        SELECT COUNT(*) FROM topics
+      SQL
+    end
+
+    def items
+      @source_db.query <<~SQL
+        SELECT * FROM topics
```

Comment:
Depending on how detailed we want the import to be, it might make sense to include everything, but we could also exclude some subtypes here. Though, I'm not sure yet, still reviewing. What do you think?